src/components/Settings/preferences/accordions/EarlyAdopterAccordion/EarlyAdopterAccordion.tsx

```diff
@@ -1,145 +0,0 @@
-import React, { ReactElement } from 'react';
-import { Checkbox, FormControlLabel, FormGroup } from '@mui/material';
-import { Formik } from 'formik';
-import { useSnackbar } from 'notistack';
-import { useTranslation } from 'react-i18next';
-import * as yup from 'yup';
-import { AccordionItem } from 'src/components/Shared/Forms/Accordions/AccordionItem';
-import { FieldWrapper } from 'src/components/Shared/Forms/FieldWrapper';
-import { FormWrapper } from 'src/components/Shared/Forms/FormWrapper';
-import { useUserPreferenceContext } from 'src/components/User/Preferences/UserPreferenceProvider';
-import { AccountListSettingsInput } from 'src/graphql/types.generated';
-import useGetAppSettings from 'src/hooks/useGetAppSettings';
-import { useUpdateAccountPreferencesMutation } from '../UpdateAccountPreferences.generated';
-
-const accountPreferencesSchema: yup.SchemaOf<
-  Pick<AccountListSettingsInput, 'tester'>
-> = yup.object({
-  tester: yup.boolean().required(),
-});
-
-interface EarlyAdopterAccordionProps {
-  handleAccordionChange: (panel: string) => void;
-  expandedPanel: string;
-  tester: boolean;
-  accountListId: string;
-  disabled?: boolean;
-}
-
-export const EarlyAdopterAccordion: React.FC<EarlyAdopterAccordionProps> = ({
```

Comment:
@dr-bizz Don't we still want the early adopter accordion, just not have the redirect? I guess we could wait to add it back until we have something we want early adopters to test.

Reply:
I guess you are correct, maybe just comment out the redirect for now and leave it there for now.

Reply:
Thanks for pointing this out though! I had forgotten about this redirect.